llmfoundry/tokenizers/tiktoken.py

```diff
@@ -90,7 +90,11 @@ def is_fast(self) -> bool:
         return False
 
     def get_vocab(self) -> Dict[str, int]:
-        """Returns vocab as a dict."""
+        """Returns vocab as a dict.
+
+        Note: This function does not work properly due to difference in assumptions between tiktoken and Hugging Face tokenizers.
```

Comment:
I think we should print a warning here in case someone calls this, just in case

Reply:
@samhavens lmk what you think of the warning i added